packages/ethereum/contracts/src/Crypto.sol

```diff
@@ -778,19 +770,11 @@ abstract contract HoprCrypto {
    * @param DST domain separation tag, used to make protocol instantiations unique
    */
   function expand_message_xmd_keccak256_single(bytes memory message, bytes memory DST) internal pure returns (bytes32 b_1, bytes32 b_2) {
-    uint256 ell; 
```

Comment:
Same as above.

Reply:
incorrectly implemented, only applies to general implementation without fixed output length